reaper.go

```diff
@@ -140,7 +140,9 @@ func newReaper(ctx context.Context, options ...reaperOption) (*reaper, error) {
 		return nil, fmt.Errorf("listen: %w", err)
 	}
 
-	r.logger.Info("listening", fieldAddress, r.listener.Addr().String())
+	// This log message, in uppercase, is in use in different Testcontainers libraries,
+	// so it is important to keep it as is to not break the current behavior of the libraries.
+	r.logger.Info("Started client processing")
```

Comment:
Sorry @mdelapenya I think you might not have understood what I meant in my previous suggestion.

I was suggesting we change the [log entry in processClients](https://github.com/testcontainers/moby-ryuk/blob/main/reaper.go#L187) instead of this one, as that is just before the service is ready to actually accept clients instead of this one when only the listener is enabled.

I wasn't suggesting changing the content of message here.

So to be clear two options:
1. Change the message here to what you had e.g. `r.logger.Info("Started", fieldAddress, r.listener.Addr().String())`
2. Change the message in `processClients` r.logger.Info("Started client processing")

In either case should keep the output of the address as thats the only way the user can see the address that the service is listening on if a ephemeral port is requested.

Reply:
I'm really sorry @stevenh. Did read it very vertically in the middle of the conference, my fault.

Reply:
@stevenh I think `1.` is better, thanks for pointing out